package/setup.py

```diff
@@ -650,6 +643,10 @@ def long_description(readme):
               'AMBER': [
                   'netCDF4>=1.0',  # for fast AMBER writing, also needs HDF5
               ],
+              'extra_formats': [   # additional file formats
+                  'netCDF4>=1.0',  # for fast AMBER writing, also needs HDF5
+                  'h5py>=2.10',    # H5MD
```

Comment:
left at minimal version for right now

Reply:
This needs to be reflected in all the CI places:
- [ ] gh action
- [ ] azure
- [ ] Travis
- [ ] maintainer's yaml

Reply:
- Travis does not install h5py. https://github.com/MDAnalysis/mdanalysis/blob/0b3ba59d7a31d461593dfa83ef50b38f171ad9eb/.travis.yml#L27-L28
- Azure does not pin or restrict packages when h5py is installed https://github.com/MDAnalysis/mdanalysis/blob/0b3ba59d7a31d461593dfa83ef50b38f171ad9eb/azure-pipelines.yml#L84-L98 although farther down, we have packages with restrictions (such as chemfiles). I am not sure what to add where... please review

Reply:
I think it's fine, I'll give it a review once I'm back from the wilds of Edinburgh

Reply:
I had change the azure file to have the restriction and the tests passed, so at least it didn't immediately break anything.